ironfish/src/rpc/routes/multisig/createSigningPackage.ts

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { SigningCommitments, UnsignedTransaction } from '@ironfish/rust-nodejs'
+import * as yup from 'yup'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+import { RpcSigningCommitments, RpcSigningCommitmentsSchema } from './types'
+
+export type CreateSigningPackageRequest = {
+  unsignedTransaction: string
+  commitments: Array<{
+    identifier: string
```

Comment:
Note that the identifiers are currently not returned with the `commitment`from the `createSigningCommitment` RPC. Submission of the commitment by a participant requires the participant to attach the identifier to their commitment, or the coordinator to keep track of who gave each commiment.